logs/logs.go

```diff
@@ -92,7 +93,12 @@ func (l *LogAgent) Run(ctx context.Context) {
 
 	t := time.NewTicker(time.Second)
 	defer t.Stop()
+	last := time.Now()
 	for {
+		if time.Since(last) >= 1 * time.Minute {
+			last = time.Now()
+			log.Printf("I! [logagent] open file count, %v", tail.OpenFileCount())
+		}
```

Comment:
Which do you prefer:
1. Log once a minute at INFO level
2. Log once a minute at DEBUG level
3. Log once a second at DEBUG level

Please let me know!

Reply:
Consider your suggestions/comments before 1 is good by me (4 or 5 maybe overkill with atomic counter and it still useful for the customer to aware the number of monitored files every interval)

Reply:
Personally I vote 3, debug every second regardless of change. 

Firstly, and more importantly, we search for files every second. So it is theoretically possible to burst with hundreds of files and close them all in the same minute (who would do that?) but even ignoring the weird edge case, I think it's more intuitive to have the log reflect the most accurate state. Not everyone is going to need it, so that's why I'd lean towards debug (also it bloats the log file a lot). 

Secondly, and slightly related, I don't know how I feel about using the same for loop but using a different frequency in it (1s timer vs 60s custom logic). If we want to emit it minutely, I'd prefer to spawn a separate goroutine for that. Cause I think having two timers of different frequencies in the same loop is confusing and idk how deterministic Go is in choosing which one to select.

Reply:
Agreed on 1 second at DEBUG.
Will fix.